### PR TITLE
BitBox02 pages

### PR DESCRIPTION
**Pull request by jstrnbrg**
### Changes: 
- Updated `walletbitbox` for `DE, EN, IT, PT_BR, RU, SL, ES`
- Removed all other `walletbitbox` instances as they still describe the BitBox01 
- Updated BitBox02 image 
- Updated BitBox02 details (link, title etc)

### CODE REVIEW

**Comment by jstrnbrg** [comment]
just added 'ES' as well

**Comment by crwatkins** [comment]
The EN update LGTM.

**Comment by jstrnbrg** [comment]
Regarding the other translations: 
- DE: myself
- IT: colleague
- PT_BR: the Portuguese translator of our BitBoxApp
- RU: colleague 
- SL: colleague 
- ES: the Spanish translator of our BitBoxApp

**Comment by jstrnbrg** [comment]
Added `JA` and `zh_CN`

File: _wallets/bitbox.md [hunk 1]
```diff
@@ -13,9 +13,9 @@ platform:
     os:
       - name: hardware
         text: "walletbitbox"
-        link: "https://shiftcrypto.ch/"
+        link: "https://shiftcrypto.ch/bitbox02/"
         source: "https://github.com/digitalbitbox/bitbox02-firmware"
-        screenshot: "bitbox.png"
+        screenshot: "02_bb02_laptop.png"
```
Comment (wbnns) [inline _wallets/bitbox.md]: This should be: `bitbox02.png`

**Comment by wbnns** [comment]
The issue is now resolved and this PR will be merged once tests pass.

**Comment by jstrnbrg** [comment]
Thank you :)

**Pull request merged**